03_qgep_db_dss.sql

```diff
@@ -1914,6 +1916,9 @@ ALTER TABLE qgep_od.reach ADD COLUMN slope_building_plan  smallint ;
 COMMENT ON COLUMN qgep_od.reach.slope_building_plan IS 'yyy_Auf dem alten Plan eingezeichnetes Plangefälle [%o]. Nicht kontrolliert im Feld. Kann nicht für die hydraulische Berechnungen übernommen werden. Für Liegenschaftsentwässerung und Meliorationsleitungen. Darstellung als z.B. 3.5%oP auf Plänen. / Auf dem alten Plan eingezeichnetes Plangefälle [%o]. Nicht kontrolliert im Feld. Kann nicht für die hydraulische Berechnungen übernommen werden. Für Liegenschaftsentwässerung und Meliorationsleitungen. Darstellung als z.B. 3.5%oP auf Plänen. / Pente indiquée sur d''anciens plans non contrôlée [%o]. Ne peut pas être reprise pour des calculs hydrauliques. Indication pour des canalisations de biens-fonds ou d''amélioration foncière. Représentation sur de plan: 3.5‰ p';
 ALTER TABLE qgep_od.reach ADD COLUMN wall_roughness  decimal(5,2) ;
 COMMENT ON COLUMN qgep_od.reach.wall_roughness IS 'yyy Hydraulische Kenngrösse zur Beschreibung der Beschaffenheit der Kanalwandung. Beiwert für die Formeln nach Prandtl-Colebrook (ks oder kb) / Hydraulische Kenngrösse zur Beschreibung der Beschaffenheit der Kanalwandung. Beiwert für die Formeln nach Prandtl-Colebrook (ks oder kb) / Coefficient de rugosité d''après Prandtl Colebrook (ks ou kb)';
+ALTER TABLE qgep_od.reach ADD COLUMN hydraulic_load decimal(7,3) ;
```

Comment:
Could we move these statements for additonal attributes in [05_datamodel_extension](https://github.com/QGEP/datamodel/blob/master/05_data_model_extensions.sql) to not mix creation of VSA-DSS datamodel and other attributes in this file?
This is a VSA-DSS attribut but will only be introduced with VSA-DSS Release 2020 and is name "Hydr_Belastung_Ist". So my suggestion would be to name it "dss2020_hydraulic_load_current". The INTERLIS definition ist 0..1000 % - so it should not be defined as decimal(7,3) but integer(?): 

!!@ comment = "30.4.2021 neu für Mehrbelastung / Hydr_Belastung"
!!@ comment = "Prozent [%]"
Prozent1000 = 0 .. 1000;

English and French description can also be added: 
Dimensioning of the discharge divided by the normal discharge capacity of the reach [%].
Débit de dimensionnement divisé par la capacité d'écoulement normale de la conduite [%].

Reply:
Done